content/issues/2024-02-21-koji-outage.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Koji Outage"
+date: "2024-02-21T21:00:00+00:00"
+affected:
+  - Koji builds
+resolved: false
+#resolvedWhen: "2024-02-22T00:00:00+00:00"
+section: issue
+severity: notice
+---
+
+Planned Outage:
+
+Koji upgrade at 2024-02-21 21:00 UTC
+
+There will be an outage starting at 2024-02-21 21:00 UTC,
+which will last approximately 3 hours.
+
+Reason for outage:
+
+Koji will be upgraded to 1.34.0, which requires a schema update that touches many rows.
+We estimate this will take about 45minutes to complete and during that time,
+Koji will be completely offline.
+
+Package maintainers are advised to not start any long term builds before the outage.
```

Comment:
We can maybe transform this to Packit:
* You should avoid new dist-git commits (e.g. merging Packit pull-requests).
* How to retrigger if the build is missing.